### ISSUE TITLE
Crazy memory leak

### ISSUE THREAD

**Issue by lennonhead** [dabbott/react-express#34]
Hi, really enjoyed your guide, helped me get up and running quickly, however when leaving the site react.express open in safari it quickly consumes all of my available system memory.  Rather concerning. Specifically noticed on the conditional rendering section.

**Comment by chaenu** [comment]
Yes, one should definitely have a look at the disqus plugin. It's infinitely making request / loading stuff, see snapshot below taken on http://www.react.express/event_handling (and it was still loading more).

<img width="621" alt="bildschirmfoto 2017-08-15 um 14 30 40" src="https://user-images.githubusercontent.com/3105445/29316084-c6164b88-81c6-11e7-8cd0-4349281839ac.png">

If you block the initial request to the embed.js, it stops way earlier and your computer cools down again. ;)
